src/js/components/file-input.js

```diff
@@ -142,27 +142,29 @@ const removeOldPreviews = (dropTarget, instructions) => {
  * @param {HTMLElement} dropTarget - target area div that encases the input
  */
 const preventInvalidFiles = (e, fileInputEl, instructions, dropTarget) => {
-  const acceptedFiles = fileInputEl.getAttribute("accept");
+  const acceptedFilesAttr = fileInputEl.getAttribute("accept");
   dropTarget.classList.remove(INVALID_FILE_CLASS);
 
   // Runs if only specific files are accepted
-  if (acceptedFiles) {
+  if (acceptedFilesAttr) {
+    const acceptedFiles = acceptedFilesAttr.split(",");
     const errorMessage = document.createElement("div");
 
     // If multiple files are dragged, this iterates through them and look for any files that are not accepted.
     let allFilesAllowed = true;
     for (let i = 0; i < e.dataTransfer.files.length; i += 1) {
       const file = e.dataTransfer.files[i];
       if (allFilesAllowed) {
-        allFilesAllowed = file.name.indexOf(acceptedFiles);
-        if (allFilesAllowed < 0) {
-          break;
+        for (let j = 0; j < acceptedFiles.length; j += 1) {
+          const fileType = acceptedFiles[j];
+          allFilesAllowed = file.name.indexOf(fileType) > 0 || file.type === fileType;
+          if (allFilesAllowed) break;
         }
```

Comment:
There's actually a little more that needs to be done here. This actually doesn't work if the first file isn't the same type as the first file type/extension from the `accept` attribute.

Reply:
I was wrong about this. Please disregard.

Reply:
managed to work on my end - attribute is `accept=".pdf,.txt"`
<img width="512" alt="Screen Shot 2020-11-24 at 2 10 57 PM" src="https://user-images.githubusercontent.com/1094951/100140546-1724f180-2e5f-11eb-99e3-f3286e62194d.png">
<img width="515" alt="Screen Shot 2020-11-24 at 2 11 30 PM" src="https://user-images.githubusercontent.com/1094951/100140548-17bd8800-2e5f-11eb-94a2-bd84405db248.png">